libs/vkd3d/shaders/cs_resolve_binary_queries.comp

```diff
@@ -1,26 +1,30 @@
 #version 450
 
+#extension GL_EXT_buffer_reference : require
+
 layout(local_size_x = 64) in;
 
-layout(binding = 0, std430)
-buffer s_query_buffer_t {
+layout(buffer_reference, std430, buffer_reference_align = 8)
+buffer query_buffer_t {
   uvec2 data[];
-} s_query_buffer;
+};
 
 layout(push_constant)
 uniform u_info_t {
-  uint query_offset;
+  query_buffer_t src_queries;
+  query_buffer_t dst_queries;
   uint query_count;
-} u_info;
+};
```

Comment:
Sure this compiles? push_constant must have an instance name.

Reply:
glslang isn't complaining. Where does it say that push_constant needs a name (and what's the reason for that anyway, given that ubo doesn't need one)?

Reply:
hm ... ye, Vulkan GLSL spec says it's optional, I can swear it used to be required ... Anyways, guess this is a non-issue then if it works in glslang.

Reply:
Ah, actually this requirement was removed three years ago, see https://github.com/KhronosGroup/Vulkan-Docs/issues/428.

Reply:
Ah, ok, so I'm not insane after all :D